scripts/tests.sh

```diff
@@ -27,3 +27,12 @@ pushd $MYTMPDIR/elm-todomvc
   nix-build
   checkfile ./result/Main.html
 popd
+
+# Library example
+pushd test/lib
+  elm2nix init > default.nix
+  elm2nix convert > elm-srcs.nix
+  elm2nix snapshot > versions.dat
```

Comment:
given the limitations, it might be a good idea to commit this file instead.

Reply:
@domenkozar just want to highlight this :point_up: In my opinion this is the most real blocker for merge now. If new `elm/core` gets published this generates newer `versions.dat` and elm compiler starts fetching lib in `elm make` phase (installPhase) which would make tests broken for people with `useSandbox = true`

Reply:
Sounds good to commit this :)

Reply:
this applies only in if we won't bring the whole solver and instead download the lowest version from the range.

Reply:
I think this is the last missing bit, we need to commit elm-srcs.nix and assert the output